docs/supported-technologies.asciidoc

```diff
@@ -197,6 +197,11 @@ The spans are named after the schema `<method> <host>`, for example `GET elastic
  inherently support context propagation as they are using `HttpUrlConnection` underneath.
 |1.4.0
 
+|Asynchronous Http Client
+|
+|
+| 2.8.1
```

Comment:
And this one is for the agent version where the support for the library has been released
```suggestion
| 1.7.0
```